docs/en_US/NNICTLDOC.md

```diff
@@ -47,13 +48,34 @@ nnictl support commands:
   |--port, -p|False| |the port of restful server|
   |--debug, -d|False||set debug mode|
 
+  * Examples
+
+    > create a new experiment with the default port: 8080
+
+    ```bash
+    nnictl create --config nni/examples/trials/mnist/config.yml
+    ```
+
+    > create a new experiment with specified port of restful server
+
+    ```bash
+    nnictl create --config nni/examples/trials/mnist/config.yml --port $8088
```

Comment:
why '$8088' instead of '8088' ?

Reply:
In a regular expression "$" means replaceable, let the user know that this value is replaceable.

Reply:
'8088' is a literal value, it is weird to be used as a variable,  either use plain literal value '8088' ,  or use [PORT] here to imply it is a replacable variable to be consistent with other parts.

Reply:
Remove "$", now use 8088 as an example, and use [experiment_id] to refer a replaceable value.